src/audio_worklet.js

```diff
@@ -168,15 +168,19 @@ class BootstrapMessages extends AudioWorkletProcessor {
         }
 #endif
         // Register a real AudioWorkletProcessor that will actually do audio processing.
-        registerProcessor(d['_wpn'], createWasmAudioWorkletProcessor(d['audioParams']));
+        // 'ap' being the audio params
+        registerProcessor(d['_wpn'], createWasmAudioWorkletProcessor(d['ap']));
 #if WEBAUDIO_DEBUG
-        console.log(`Registered a new WasmAudioWorkletProcessor "${d['_wpn']}" with AudioParams: ${d['audioParams']}`);
+        console.log(`Registered a new WasmAudioWorkletProcessor "${d['_wpn']}" with AudioParams: ${d['ap']}`);
 #endif
         // Post a Wasm Call message back telling that we have now registered the
         // AudioWorkletProcessor class, and should trigger the user onSuccess
         // callback of the
         // emscripten_create_wasm_audio_worklet_processor_async() call.
-        p.postMessage({'_wsc': d['callback'], 'x': [d['contextHandle'], 1/*EM_TRUE*/, d['userData']] }); // "WaSm Call"
+        // 'cb' is the callback function
+        // 'ch' the context handle
+        // 'ud' the passed user data
+        p.postMessage({'_wsc': d['cb'], 'x': [d['ch'], 1/*EM_TRUE*/, d['ud']] }); // "WaSm Call"
```

Comment:
Isn't `_wpn` part of the message name so could clash? Same as `_wpn`, the worklet processor name.

There was already an existing `x` in there so I didn't add underscores.

I did tidy the comments though because they repeat the same thing.

Reply:
> @juj do you remember why you add the underscore?

Commented in https://github.com/emscripten-core/emscripten/pull/22472#discussion_r1752319062